### PR TITLE
Added parser for "/bin/ls -lARkhvX /lib/firmware"

### PR DESCRIPTION
**Pull request by vishwanathjadhav**

### CODE REVIEW

**Comment by vishwanathjadhav** [comment]
Jira Ticket:
https://projects.engineering.redhat.com/browse/INSPEC-82

**Comment by vishwanathjadhav** [comment]
Specs Changes: https://github.com/RedHatInsights/insights-frontend-assets/pull/153

**Comment by bfahr** [comment]
@vishwanathjadhav I believe this parser could benefit from the changes being added in PR #1360.  @csams what do you think?

File: insights/specs/__init__.py [hunk 1]
```diff
@@ -215,6 +215,7 @@ class Specs(SpecSet):
     lspci_kernel = RegistryPoint()
     lssap = RegistryPoint()
     lsscsi = RegistryPoint()
+    ls_lib_fw = RegistryPoint()
```
Comment (bfahr) [inline insights/specs/__init__.py]: @vishwanathjadhav sorry about not asking this before you pushed your last commit, but wouldn't it be clearer to name the spec `ls_lib_firmware` to be consistent with the parser name and with the `ls_sys_firmware` spec?

Comment (vishwanathjadhav) [reply insights/specs/__init__.py]: Done

**Pull request merged**